pymlst/2.1.5/Dockerfile

```diff
@@ -0,0 +1,82 @@
+## Builder ##
+FROM ubuntu:jammy as blat
+
+ARG BLAT_VER="35"
+ARG MACHTYPE="x86_64-pc-linux-gnu"
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    wget \
+    unzip \
+    libpng-dev
+
+RUN wget https://genome-test.gi.ucsc.edu/~kent/src/blatSrc${BLAT_VER}.zip &&\
+    unzip blatSrc${BLAT_VER}.zip  &&\
+    cd blatSrc &&\
+    mkdir -p bin/${MACHTYPE} &&\
+    mkdir -p lib/${MACHTYPE} &&\
+    make HOME=$PWD MACHTYPE=${MACHTYPE} C_INCLUDE_PATH=/usr/include LIBRARY_PATH=/usr/lib CFLAGS="-fcommon" &&\
+    mv bin/${MACHTYPE}/* /usr/local/bin/
+
+## App ##
+FROM ubuntu:jammy as app 
+
+ARG PYMLST_VER="2.1.5"
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="pyMLST"
+LABEL software.version="${PYMLST_VER}"
+LABEL description="A Python Mlst Local Search Tool"
+LABEL website="https://github.com/bvalot/pyMLST"
+LABEL license="https://github.com/bvalot/pyMLST/blob/master/LICENSE"
+LABEL maintainer="Kutluhan Incekara"
+LABEL maintainer.email="kutluhan.incekara@ct.gov"
+
+#install dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y\
+    mafft \
+    kma \
+    python3-pip \
+    python3-biopython &&\
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# get blat from builder
+COPY --from=blat /usr/local/bin/* /usr/local/bin/
+
+# install pymlst
+RUN pip3 install --no-cache pymlst==${PYMLST_VER}
+
+CMD ["pyMLST", "-h"]
```

Comment:
Is 'pyMLST' actually executable? All the examples I'm seeing use `claMLST` or `wgMLST`.

Reply:
You are right! It seems pyMLST is for configuration and log level which is not so meaningful alone.